kotlin-jvm/src/main/kotlin/playground/_main.kt

```diff
@@ -23,6 +23,7 @@ fun main() {
     playground.okio.main()
     playground.retrofit.main()
     playground.sqldelight.main()
+    playground.apollo.main()
```

Comment:
please keep this list sorted, it helps to avoid merge conflicts when multiple PR are merged